interface/main/main_screen.php

```diff
@@ -15,14 +15,169 @@
  *
  * @package OpenEMR
  * @author  Brady Miller <brady.g.miller@gmail.com>
+ * @author  Rod Roark <rod@sunsetsystems.com>
  * @link    http://www.open-emr.org
  */
 
+/* Include our required headers */
+require_once('../globals.php');
 
+use OpenEMR\Core\Header;
```

Comment:
Also place the other Classes that are used in this script (such as u2flib_server\U2F )

Reply:
Even if code references the long name? In that case it doesn't require the shortcut.

Reply:
It makes it much easier to track down where things are used in the codebase.

Reply:
(plus it gives OpenEMR that cool, "we use classes and are modern" feel :) )